layouts/community/ergodox/xyverz/keymap.c

```diff
@@ -14,36 +15,36 @@
  */
 
 #include QMK_KEYBOARD_H
-#include "debug.h"
-#include "action_layer.h"
-#include "eeconfig.h"
 
-extern keymap_config_t keymap_config;
+enum layer_names {
+    _DVORAK,
+    _QWERTY,
+    _COLEMAK,
+    _MEDIA,
+};
+
+enum custom_keycodes { DVORAK = SAFE_RANGE, QWERTY, COLEMAK, MEDIA };
```

Comment:
```suggestion
enum custom_keycodes { DVORAK = SAFE_RANGE, QWERTY, COLEMAK };
```

Reply:
I've made this change. I've also added more to the comments and updated the readme.md.